Breeding-Insight/bi-web
# [BI-1445] - Trait Entity/Attribute/Method capitalization

# Description
**Story:** [BI-1445 - Trait Column value not the same as entered in the the Term creation](https://breedinginsight.atlassian.net/browse/BI-1445)

Implemented changes to make capitalization of trait entity, trait attribute, and method description more consistent in table and sidepanel display.

For display in table and details panel, the first letter of each word for the trait entity, trait attribute, and method description will be capitalized
For trait attribute and method description the rest of the capitalization will match what the user entered, and on edit the user will see the capitalization as they entered it.
For trait entity, due to it being a set and therefore having to have consistent case, regardless of user entry for capitalization the first letter of each word will always be capitalized and the rest of the letters will be lowercase in both display and in edit.

- Added display-case to main.scss which makes the first letter of each word capitalized
- Utilized display-case instead of the custom capitalize function in ontology table and detail panel
- Modified TraitService:getAttributesEntitiesDescriptions and OntologyTable:AttributesEntitiesDescriptions to use a large default page size so all entries will be retrieved for the autocomplete (NOTE: retrieving and using the total count of ontology terms should be implemented in a future change)

# Dependencies
[bi-api/BI-1445](https://github.com/Breeding-Insight/bi-api/pull/189)

# Testing
- Open ontology table
- Create new ontology term with multiple words and varied capitalization
- For new term and existing terms:
- Check that the first letter of each word in trait and method columns in table is capitalized
- Check that the first letter of each word for trait entity, trait attribute, and method description in detail panel is capitalized
- For table and detail panel check that each word in trait entity is lowercase apart from the first letter
- Open edit for ontology term, check that trait entity is all lowercase except for first letter of each word, and that trait attribute and method description are capitalized as user entered

# Checklist:

- [ ] I have run TAF: [_\<link to TAF run>_](https://github.com/Breeding-Insight/taf/actions/runs/2471799460)
